spec/imsc-hrm.html

```diff
@@ -641,7 +641,7 @@ <h2>Paint Text</h2>
           </tr>
 
           <tr>
-            <td>latin, greek, cyrillic, hebrew or base</td>
+            <td><code>latin</code>, <code>greek</code>, <code>cyrillic</code>, <code>hebrew</code> or <code>common</code></td>
```

Comment:
Nit: generally the script names are presented in title case (e.g. `Latin`, `Common`, etc.)

Reply:
Addressed at https://github.com/w3c/imsc-hrm/pull/45/commits/4576093955abcc77818147578d8f651b1c3c6a0d